requests_toolbelt/downloadutils/stream.py

```diff
@@ -12,6 +12,7 @@
     r';\s*(%s|[^\s;=]+)\s*(?:=\s*(%s|[^;]+))?\s*' % (_QUOTED_STRING_RE,
                                                      _QUOTED_STRING_RE)
 )
+_DEFAULT_CHUNKSIZE = 65536
```

Comment:
logic: 512 bytes is very low for most download where you care about streaming to a file, and `tee` already uses 64KB.